app/javascript/gobierto_data/webapp/components/commons/Info.vue

```diff
@@ -2,51 +2,50 @@
       <div class="gobierto-data-summary-header-container">
-        <i
-          class="fas fa-clock"
-          style="color: var(--color-base); opacity: .25"
-        />
-        <span class="gobierto-data-summary-header-container-label">
-          {{ labelUpdated }}
-        </span>
-        <span
-          v-if="dateUpdated"
-          class="gobierto-data-summary-header-container-text"
-        >
-          {{ dateUpdated | convertDate }}
-        </span>
-      </div>
-      <div class="gobierto-data-summary-header-container">
-        <i
-          class="fas fa-calendar"
-          style="color: var(--color-base); opacity: .25"
-        />
-        <span class="gobierto-data-summary-header-container-label">
-          {{ labelFrequency }}
-        </span>
-        <span
-          v-if="frequencyDataset"
-          class="gobierto-data-summary-header-container-text"
-        >
-          {{ frequencyDataset }}
-        </span>
-      </div>
-      <div class="gobierto-data-summary-header-container">
-        <i
-          class="fas fa-tag"
-          style="color: var(--color-base); opacity: .25"
-        />
-        <span class="gobierto-data-summary-header-container-label">
-          {{ labelSubject }}
-        </span>
-        <a
-          v-if="categoryDataset"
-          href=""
-          class="gobierto-data-summary-header-container-text-link"
-        >
-          {{ categoryDataset }}
-        </a>
+        <template v-if="dateUpdated">
+          <i
+            class="fas fa-clock"
+            style="color: var(--color-base); opacity: .25"
+          />
+          <span class="gobierto-data-summary-header-container-label">
+            {{ labelUpdated }}
+          </span>
+          <span class="gobierto-data-summary-header-container-text">
+            {{ dateUpdated | convertDate }}
+          </span>
+        </template>
       </div>
+      <template v-if="frequencyDataset">
+        <div class="gobierto-data-summary-header-container">
+          <i
+            class="fas fa-calendar"
+            style="color: var(--color-base); opacity: .25"
+          />
+          <span class="gobierto-data-summary-header-container-label">
+            {{ labelFrequency }}
+          </span>
+          <span class="gobierto-data-summary-header-container-text">
+            {{ frequencyDataset }}
+          </span>
+        </div>
+      </template>
+      <template v-if="categoryDataset">
+        <div class="gobierto-data-summary-header-container">
+          <i
+            class="fas fa-tag"
+            style="color: var(--color-base); opacity: .25"
+          />
+          <span class="gobierto-data-summary-header-container-label">
+            {{ labelSubject }}
+          </span>
+          <a
+            href="#"
+            class="gobierto-data-summary-header-container-text-link"
+          >
+            {{ categoryDataset }}
+          </a>
+        </div>
+      </template>
```

Comment:
You see you're using almost the same markup, right? Do you know how do we call stuff like that, right? Yeah, component

Reply:
3rd time rule: "if you need to copy paste the same thing 3 times, you're doing something wrong"

Reply:
you're DRY